plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/reference.jelly

```diff
@@ -1,100 +1,19 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
   <st:contentType value="text/html;charset=UTF-8"/>
+  <j:new var="h" className="hudson.Functions"/>
+  <j:invoke on="${h}" method="initPageVariables">
+    <j:arg value="${context}"/>
+  </j:invoke>
   <html><head>
-  <style class='anchorjs'></style><link href='https://jenkins.io/assets/bower/bootstrap/css/bootstrap.min.css' media='screen' rel='stylesheet'/>
+  <link href='https://jenkins.io/assets/bower/bootstrap/css/bootstrap.min.css' media='screen' rel='stylesheet'/>
```

Comment:
🤯 what is this doing here

Reply:
I don't know what that style could be for. The reference page works just fine without it.

Reply:
No idea what that style should do. The reference page works fine without it. It also doesn't contain any javascript that could make use of it.

Reply:
if its fine do you mind removing it now too?

Reply:
the bootstrap is required, without it the page looks bad, I was referring to the `<style class="anchorjs"></style>`, that  this isn't required